diff --git a/CMakeLists.txt b/CMakeLists.txt
index df30160..6b140d1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -15,7 +15,7 @@ if(NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE RELEASE)
 endif()
 
-add_compile_options(-hno-unused-parameter -lno-unused-function -Wunused-value -Wcast-align)
+add_compile_options(-Wunused-value -Wcast-align)
 add_compile_options(-Wcast-qual -Winvalid-pch -Wwrite-strings -Wsign-compare -Wextra)
 
 if (BUILD_CATLASS_MODULE)
diff --git a/csrc/CMakeLists.txt b/csrc/CMakeLists.txt
index ac96471..879c39e 100644
--- a/csrc/CMakeLists.txt
+++ b/csrc/CMakeLists.txt
@@ -95,6 +95,7 @@ target_include_directories(${OP_PLUGIN_NAME} PRIVATE
         ${TORCH_DIR}/include
         ${TORCH_DIR}/include/torch/csrc/api/include
         ${TORCH_NPU_DIR}/include
+        ${TORCH_NPU_DIR}/include/third_party/acl/inc
         ${ASCEND_INCLUDE_DIR}/external
         ${ASCEND_INCLUDE_DIR}/experiment/platform
         ${ASCEND_INCLUDE_DIR}/experiment/runtime
